other/bootstrap_daemon/tox-bootstrapd.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=-mkdir /var/run/tox-bootstrapd -p
+ExecStartPre=chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd
+WorkingDirectory=/var/lib/tox-bootstrapd
+ExecStart=/usr/bin/tox-bootstrapd /etc/tox-bootstrapd.conf
+User=tox-bootstrapd
+Group=tox-bootstrapd
+PIDFile=/var/run/tox-bootstrapd.pid
+#CapabilityBoundingSet=CAP_NET_BIND_SERVICE
```

Comment:
It should be noted that on many systems this requires an external package like ``libcap2-bin``.

Also, no equivalent instructions for sysvinit/emulated machines are provided.

Final note, the user should be warned beforehand that the daemon will attempt to bind on 443 with this setup, something that will conflict with the use of https servers.